packages/components/molecules/f-mega-modal/src/components/MegaModal.vue

```diff
@@ -33,7 +34,10 @@
                         :is="titleHtmlTag"
                         v-if="title"
                         :id="uid"
-                        :class="['c-megaModal-title', $style['c-megaModal-title']]"
+                        :class="['c-megaModal-title', $style['c-megaModal-title'], {
+                            [$style['c-megaModal-hasBackBtn']]: hasBackButton,
```

Comment:
I would put these modifiers up onto the `c-megaModal` `div` element. The modifier class should then be `c-megaModal--hasBackBtn` too.

Just means if any further styling modifications are needed, you can style it so that

```scss
.c-megaModal--hasBackBtn {
  & .c-megaModal-title { … }
  … // any other stuff here too
}